src/Codeception/Lib/InnerBrowser.php

```diff
@@ -190,13 +190,21 @@ protected function clientRequest($method, $uri, array $parameters = [], array $f
             }
         }
 
-        if (!ReflectionHelper::readPrivateProperty($this->client, 'followRedirects')) {
+        if (method_exists($this->client, 'isFollowingRedirects')) {
+            $isFollowingRedirects = $this->client->isFollowingRedirects();
+            $maxRedirects = $this->client->getMaxRedirects();
+        } else {
+            //Symfony 2.7 support
+            $isFollowingRedirects = ReflectionHelper::readPrivateProperty($this->client, 'followRedirects');
+            $maxRedirects = ReflectionHelper::readPrivateProperty($this->client, 'maxRedirects', 'Symfony\Component\BrowserKit\Client');
```

Comment:
Is the class name necessary in this line? Both lines should be consistent 
Everything else is perfect.

Note to myself: make Travis run one build with lowest versions of all dependencies again.

Reply:
you are right, class name removed